test/LoopFuserTest.cxx

```diff
@@ -88,7 +89,11 @@ CUDA_TEST(TestPacker, packFixedRange) {
 
 #ifdef __CUDACC__
    cudaDeviceSynchronize();
```

Comment:
Could you use `gpuDeviceSynchronize();` here?

Reply:
Throughout this file.

Reply:
Done